tutorials/integrate_3rd_libs_into_pipeline/metadata.json

```
@@ -0,0 +1,50 @@
+{
+	"tutorial": {
+		"name": "Integrate 3rd party libaries into Holoscan pipeline",
+		"description": "General tutorials on integrate 3rd party libaries into Holoscan SDK based applications / pipelines",
+		"authors": [
+			{
+				"name": "Meiran Peng",
+				"affiliation": "NVIDIA"
+			}
+		],
+        "language": "Python",
+		"version": "0.1.0",
+		"changelog": {
+			"0.1.0": "Initial Release"
+		},
+		"holoscan_sdk": {
+			"minimum_required_version": "0.6.0",
+			"tested_versions": [
+				"0.6.0"
+			]
```

Can you confirm what Holoscan SDK version was used for development? The latest release (before today) was v1.0.3:

```suggestion
			"minimum_required_version": "1.0.3",
			"tested_versions": [
				"1.0.3"
			]
```

---

I tested this integration for all the libraries are in 0.6.0.

---

Okay, thanks for confirming.